packages/admin-frontend/pages/featured.js

```diff
@@ -0,0 +1,317 @@
+// @flow
+/**
+ * Part of GDL gdl-frontend.
+ * Copyright (C) 2018 GDL
+ *
+ * See LICENSE
+ */
+
+import * as React from 'react';
+import Select from '@material-ui/core/Select/Select';
+import Button from '@material-ui/core/Button/Button';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import InputLabel from '@material-ui/core/InputLabel/InputLabel';
+import FormControl from '@material-ui/core/FormControl/FormControl';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import { Form, Field, FormSpy } from 'react-final-form';
+
+import {
+  fetchLanguages,
+  fetchFeaturedContent,
+  updateFeaturedContent,
+  saveFeaturedContent,
+  deleteFeaturedContent
+} from '../lib/fetch';
+import withMuiRoot from '../withMuiRoot';
+import Container from '../components/Container';
+import type { FeaturedContent, Language } from '../types';
+
+type State = {
+  featuredContent: ?FeaturedContent,
+  selectedLanguage: string,
+  defaultReturned: boolean
+};
+
+class EditFeaturedContent extends React.Component<
+  { languages: Array<Language> },
+  State
+> {
+  static async getInitialProps() {
+    const languagesRes = await fetchLanguages();
+
+    return {
+      languages: languagesRes.isOk ? languagesRes.data : []
+    };
+  }
+
+  state = {
+    featuredContent: null,
+    selectedLanguage: '',
+    defaultReturned: true
```

Comment:
I think we can drop `defaultReturned` from the state.
Isn't it possible to compute this by comparing the other two values in the component state?